### DISCUSSION TITLE
Adapters not working with extension cords (required for input viewer use)

### DISCUSSION THREAD

**Discussion by SoulSwordStrike** [darthcloud/BlueRetro#835] [Q&A] [answered]
Hello, really enjoying the blueretro project and devices. Just wanted to see if there was any way to make the adapters work while using extension cords? This is primarily needed for use with an input viewer. Since the viewer is a box in the middle of two extension cords. I have even tried the adapters without the viewer and only the extension cord and it doesnt work so I know its not the viewer box so far. 
I just get a solid light and no response, and no change when using the button on the adapter. The adapters work fine in my controller ports directly. I have tried this on saturn and n64 so far, I would imagine behavior would be similar on other adapters for other machines though. Thank you for any help on this.

**Comment by darthcloud** [comment, marked as answer]
That's not something I test for.

I doubt the problem is using an extension cord, but that always possible.

I think it's more likely that your viewer interfere with the data line.
The ESP32 is much more sensitive to reflexion than regular controllers.

The real solution would be for the viewer data to be taken from the esp32 directly.
But that would need some work, so probably not anytime soon.
